test/built-ins/Temporal/PlainYearMonth/prototype/add/custom-daysInMonth-irrelevant.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2023 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal-adddurationtoorsubtractdurationfromplainyearmonth
+description: Addition of a negative duration to a PlainYearMonth is not influenced by the implementation of daysInMonth()
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+class CustomCalendar extends Temporal.Calendar {
+  constructor() {
+    super("iso8601");
+  }
+  daysInMonth(ym, ...args) {
+      console.log(ym)
+      if (ym.month == 3) {
+          return 15;
+      }
+      return super.daysInMonth(...arguments);
```

Comment:
To make it simpler, you can just make a completely broken `daysInMonth`:
```suggestion
    return 15;
```